tests/Container/UtilTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Illuminate\Tests\Container;
+
+use Illuminate\Container\Util;
+use PHPUnit\Framework\TestCase;
+
+class UtilTest extends TestCase
+{
+    public function testUnwrapIfClosure()
+    {
+        $this->assertSame('foo', Util::unwrapIfClosure('foo'));
+        $this->assertSame('foo', UtiL::unwrapIfClosure(function () {
```

Comment:
@iansltx I noticed a capital `L` on this line.

// @driesvints

Reply:
Whoops.

Mind PR'ing a typo fix here, as this has already been merged?